src/main/java/lotto/Validate.java

```diff
@@ -0,0 +1,47 @@
+package lotto;
+
+import java.util.List;
+
+public class Validate {
+
+    static void validate_Size(List<Integer> numbers){
```

Comment:
자바의 메서드 네이밍 컨벤션은 lowerCamelCase 입니다!
```suggestion
    static void validateSize(List<Integer> numbers){
```